kubeflow/common/knative/Makefile

```diff
@@ -0,0 +1,23 @@
+# The name of the context for your Kubeflow cluster
+# NAME=$(shell yq r ./Kptfile 'openAPI.definitions."io.k8s.cli.setters.name".x-k8s-cli.setter.value')
+# KFCTXT=$(NAME)
+
+.PHONY: apply
+apply:
+# Common Knative
+# It has error when trying to patch the resource downloaded from knative because it doesn't match kustomize requirement.
+# Original release of knative manifest can not be patched by kustomize because of `already registered id`. For now we run kubectl directly.
```

Comment:
this typically means you are combining two packages that have duplicate resources,
a simpler workaround is to figure out what is duplicate and break down apply to multiple steps.

but we can optimize this in a separate PR

Reply:
Ack, thank you for the info! Currently I am using David Spek's approach to breakdown the duplicated items and adjust to kustomize build rule. We can further optimize it later.